src/pages/question/question.css

```diff
@@ -18,13 +18,13 @@
 }
 
 .option-item {
-    color: white;
+    /* color: white; */
```

Comment:
You can remove this comment if not needed

Reply:
Done, thanks!